bigbite/Jake-Week-2
# Feat/homepage

## Description
Created a Home page for a Mental Health Awareness website that includes; A navigation bar, Introductory paragraph, banner containing 3 buttons (that in further commits will open 3 separate modals accordingly), a list with hover functionality and a footer. Created colour scheme variables and global classes such as 'regular-btn' that can be used on further pages linking to the Home page.


## Change Log



- Initial file setup alongside Global CSS Variables.
- Created Nav bar with hover functionality.
- Created Introduction section and in turn added a new font for h1,h2,h3.
- Created a banner with 3 containers, each containing a button. 
- Created a list with an image corresponding to a title and short description.
- Created a footer, used ASCII for the copyright symbol.
- Added hover functionality to banner containers and list items, in which background colour changes alongside a scale transformation.

## Steps to test


To test the hover functionality, hover over any list item within the 'tips' section and ensure it scales up and changes background colour. 
To test hover functionality for 'the benefits of mental health awareness' section, ensure the container scales up when hovered over. 
## Screenshots/Videos

https://github.com/bigbite/Jake-Week-2/assets/53060167/40352e24-25fb-4b4b-8aa2-5d6f8b1bdbc9




## Checklist:
